woocommerce/woocommerce-ios
# Mobile payments support for dynamic type in the payment success modal

Closes: #5519


###### Merge after the target branch PR

### Testing instructions

* Make an IPP
* Change font size via accessibility and check that it's all readable on the successful IPP state

### Screenshots

<img width="287" alt="image" src="https://user-images.githubusercontent.com/4923871/169283711-7439b857-2af7-479e-bc67-57b9f5452cdb.png">
<img width="282" alt="image" src="https://user-images.githubusercontent.com/4923871/169283790-c03bc982-6e25-4a6a-8fbe-54c755ad54a9.png">
<img width="290" alt="image" src="https://user-images.githubusercontent.com/4923871/169283844-1f9a3fd8-3b76-4d8c-9837-7e8c74da6aa9.png">


---
